make.xsh

```diff
@@ -133,30 +133,55 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
     Use the "database-backup" service in the "docker-compose.yml" file to backup
     all of the tables in the instace of Postgres running locally.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
+    docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+
+@makefile.target(tag="run services locally")
+def backup_local_db():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally. You must have a
+    postgres container running which you can run using "make run-local-db"
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
```

Comment:
Fancy :D

Reply:
Why are we specifying the `POSTGRES_PORT` here? Is that not already in the `docker-compse.yml`? I feel like there would be a better place for it. I think it'd be useful to expose that port in all cases.

Reply:
more copy and paste with a bit of my own panache but I felt kind of proud of this

Reply:
Can you remove the `-p $POSTGRES_PORT:$POSTGRES_PORT` part and have the `docker-compose.yml` take care of that?